docs/autogenerated_docs/4.1-members-api.md

```
@@ -0,0 +1,689 @@
+| `/api/create-stripe-setup-session` | POST | Create session to set up payment method |
+| `/webhooks/stripe` | POST | Webhook handler for Stripe events |
+
+Sources:
+- [ghost/core/core/server/web/members/app.js:85-95](https://github.com/TryGhost/Ghost/blob/0d0e5bd3/ghost/core/core/server/web/members/app.js#L85-L95)
+- [ghost/core/core/server/web/members/app.js:38](https://github.com/TryGhost/Ghost/blob/0d0e5bd3/ghost/core/core/server/web/members/app.js#L38)
+
+### Create Stripe Checkout Session
+
+This endpoint creates a Stripe checkout session for members to purchase a subscription.
+
+**Path:** `/api/create-stripe-checkout-session/`
```

markdownlint-cli2 (0.18.1)

Strong style
Expected: underscore; Actual: asterisk

(MD050, strong-style)